tests/graphql_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,54 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`argument_comment.graphql 1`] = `
+
+query (
+  $string: String, # Some variable comment
+  $bool: Boolean # Some comment
+ ) {
+   someField
+ }
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+query(
+  # Some variable comment
+  $string: String,
```

Comment:
Does this comment go through your special case? It should be kept at the end of the line like it was in the original source.